package.json

```diff
@@ -368,11 +368,11 @@
     ]
   },
   "resolutions": {
-    "@storybook/react-docgen-typescript-plugin": "1.0.6--canary.9.0c3f3b7.0",
```

Comment:
storybook now uses this so we no longer need this override https://github.com/storybookjs/storybook/blob/bbfdf5897e399f65368ca2ed14930b912e466a25/code/presets/react-webpack/package.json#L69